.hound.yml

```diff
@@ -5,3 +5,7 @@ eslint:
 javscript:
   enabled: true
   config_file: .jshintrc
+
+  tslint:
+  enabled: true
```

Comment:
```tslint:``` is not indented correctly. Hound's own file should not have any suggestions for itself. Otherwise, it looks good 👍

Reply:
Oops! Didn't see that. fixing it right away.

Reply:
> Oops! Didn't see that. fixing it right away.

@gittysachin fixed the indentation.